cmd/root.go

```diff
@@ -5,12 +5,16 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
+var EnvPrefix = "ASTRIA_GO"
+
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
-	Use:   "astria-go",
-	Short: "A CLI to run Astria, interact with the Sequencer, deploy rollups, and more.",
+	Use:              "astria-go",
+	Short:            "A CLI to run Astria, interact with the Sequencer, deploy rollups, and more.",
+	PersistentPreRun: validateLogLevels,
```

Comment:
I think this would get overridden if a subcommand defined `PersistentPreRun`, but if we use `PreRun` here it will always get ran even when a subcommand defines a `PreRun` or `PersistentPreRun`

Reply:
I dug into this a little more and found that we only need to use the `PersistentPreRun` on the root command to set the log level. The `PersistentPreRun` of all commands in the command tree will always be run in order of root -> descending parents -> current command, and then only the `PreRun` of the current command would get run. Because we want to set the log level at root we only need the `PersistentPreRun` there. I was also able to remove the `PreRun`s for setting the logs on all the sub commands now that root `PersistentPreRun` is set up.